### ISSUE TITLE
Issues in installing step 6 "python main.py"

### ISSUE THREAD

**Issue by asdxzsa** [douglaslab/cadnano2#4]
Hi, I have a trouble in installing step 6.
I followed each steps and at step 6, I typed  "python main.py" and this happened.

<img width="722" alt="Error" src="https://user-images.githubusercontent.com/39143997/71337493-4cf69180-258f-11ea-8dec-00e427f5e92a.png">

I don't know how to solve this problem. Any comments and helps will be grateful.
Thank you!

**Comment by eGFlopp** [comment]
Hi,

I just tried installing for the first time and I have the same error showing up. I tried a few times and get this output:

python main.py
Traceback (most recent call last):
  File "main.py", line 17, in <module>
    cadnano.initAppWithGui()
  File ".\cadnano.py", line 43, in initAppWithGui
    from cadnanoqt import CadnanoQt
  File ".\cadnanoqt.py", line 8, in <module>
    util.qtWrapImport('QtWidgets', globals(),  ['qApp', 'QApplication', 'QUndoGroup'])
  File ".\util.py", line 103, in qtWrapImport
    assert(False)  # Have not found a suitable qt framework
AssertionError

Any fixes so far?

Thank you!
